compact_activity_snapshot.proto

```diff
@@ -64,62 +64,67 @@ message Backend {
     WAIT_EVENT_LWLOCK_REPLICATION_SLOT_CONTROL_LOCK = 137; // ReplicationSlotControlLock
-    WAIT_EVENT_LWLOCK_COMMIT_TS_CONTROL_LOCK = 138; // CommitTsControlLock
+    WAIT_EVENT_LWLOCK_COMMIT_TS_CONTROL_LOCK = 138; // CommitTsControlLock, CommitTsSLRULock
     WAIT_EVENT_LWLOCK_COMMIT_TS_LOCK = 139; // CommitTsLock
     WAIT_EVENT_LWLOCK_REPLICATION_ORIGIN_LOCK = 140; // ReplicationOriginLock
     WAIT_EVENT_LWLOCK_MULTI_XACT_TRUNCATION_LOCK = 141; // MultiXactTruncationLock
     WAIT_EVENT_LWLOCK_OLD_SNAPSHOT_TIME_MAP_LOCK = 142; // OldSnapshotTimeMapLock
-    WAIT_EVENT_LWLOCK_BACKEND_RANDOM_LOCK = 143; // BackendRandomLock
+    WAIT_EVENT_LWLOCK_BACKEND_RANDOM_LOCK = 143; // BackendRandomLock (removed in 12)
     WAIT_EVENT_LWLOCK_LOGICAL_REP_WORKER_LOCK = 144; // LogicalRepWorkerLock
-    WAIT_EVENT_LWLOCK_CLOG_TRUNCATION_LOCK = 145; // CLogTruncationLock
-    // Source: lwlock.h (BuiltinTrancheIds)
-    WAIT_EVENT_LWTRANCHE_CLOG_BUFFERS = 146; // clog
-    WAIT_EVENT_LWTRANCHE_COMMITTS_BUFFERS = 147; // commit_timestamp
-    WAIT_EVENT_LWTRANCHE_SUBTRANS_BUFFERS = 148; // subtrans
-    WAIT_EVENT_LWTRANCHE_MXACTOFFSET_BUFFERS = 149; // multixact_offset
-    WAIT_EVENT_LWTRANCHE_MXACTMEMBER_BUFFERS = 150; // multixact_member
-    WAIT_EVENT_LWTRANCHE_ASYNC_BUFFERS = 151; // async
-    WAIT_EVENT_LWTRANCHE_OLDSERXID_BUFFERS = 152; // oldserxid
-    WAIT_EVENT_LWTRANCHE_WAL_INSERT = 153; // wal_insert
-    WAIT_EVENT_LWTRANCHE_BUFFER_CONTENT = 154; // buffer_content
-    WAIT_EVENT_LWTRANCHE_BUFFER_IO_IN_PROGRESS = 155; // buffer_io
-    WAIT_EVENT_LWTRANCHE_REPLICATION_ORIGIN = 156; // replication_origin
-    WAIT_EVENT_LWTRANCHE_REPLICATION_SLOT_IO_IN_PROGRESS = 157; // replication_slot_io
-    WAIT_EVENT_LWTRANCHE_PROC = 158; // proc
-    WAIT_EVENT_LWTRANCHE_BUFFER_MAPPING = 159; // buffer_mapping
-    WAIT_EVENT_LWTRANCHE_LOCK_MANAGER = 160; // lock_manager
-    WAIT_EVENT_LWTRANCHE_PREDICATE_LOCK_MANAGER = 161; // predicate_lock_manager
-    WAIT_EVENT_LWTRANCHE_PARALLEL_HASH_JOIN = 162; // parallel_hash_join
-    WAIT_EVENT_LWTRANCHE_PARALLEL_QUERY_DSA = 163; // parallel_query_dsa
-    WAIT_EVENT_LWTRANCHE_SESSION_DSA = 164; // session_dsa
-    WAIT_EVENT_LWTRANCHE_SESSION_RECORD_TABLE = 165; // session_record_table
-    WAIT_EVENT_LWTRANCHE_SESSION_TYPMOD_TABLE = 166; // session_typmod_table
-    WAIT_EVENT_LWTRANCHE_SHARED_TUPLESTORE = 167; // shared_tuplestore
-    WAIT_EVENT_LWTRANCHE_TBM = 168; // tbm
-    WAIT_EVENT_LWTRANCHE_PARALLEL_APPEND = 169; // parallel_append
+    WAIT_EVENT_LWLOCK_CLOG_TRUNCATION_LOCK = 145; // CLogTruncationLock, XactTruncationLock
+    // (Jumping the number here as 146- is taken by LWTRANCHE)
+    WAIT_EVENT_LWLOCK_WRAP_LIMITS_VACUUM_LOCK = 180; // WrapLimitsVacuumLock
+    WAIT_EVENT_LWLOCK_NOTIFY_QUEUE_TAIL_LOCK = 181; // NotifyQueueTailLock
+    // Source: lwlock.h (BuiltinTrancheIds), lwlock.c (BuiltinTrancheNames)
+    WAIT_EVENT_LWTRANCHE_CLOG_BUFFERS = 146; // clog, XactBuffer
```

Comment:
[note for reviewer] related commits for the rename:
https://github.com/postgres/postgres/commit/5da14938f7bfb96b648ee3c47e7ea2afca5bcc4a
https://github.com/postgres/postgres/commit/36ac359d3621578cefc2156a3917024cdd3b1829